packages/lido-ui/src/index.ts

```diff
@@ -15,6 +15,8 @@ export * from '@lidofinance/lido-logo'
 export * from '@lidofinance/link'
 export * from '@lidofinance/loaders'
 export * from '@lidofinance/modal'
+export * from '@lidofinance/modal-button'
```

Comment:
We need to discuss with Stan, our designer, whether this style of button will only be used for modals only. If not, then this name might be too specific and possibly misleading.

Reply:
This task discussed with @avsetsin. We decided that this button will used for modals only.

so, we can to change the component name anytime.

I will to raise this problem on 24.08.2021 meeting.